pkg/virt-controller/watch/clone/clone_test.go

```diff
@@ -185,9 +185,6 @@ var _ = Describe("Clone", func() {
 		)
 		sourceVMI.Namespace = metav1.NamespaceDefault
 		sourceVM = libvmi.NewVirtualMachine(sourceVMI)
-		sourceVM.Spec.Running = nil
-		runStrategy := virtv1.RunStrategyHalted
-		sourceVM.Spec.RunStrategy = &runStrategy
```

Comment:
Don't we need to set the RunStrategy to `Halted`. Is `nil` equivalent?

Reply:
It is already done in `sourceVM = libvmi.NewVirtualMachine(sourceVMI)`